### DISCUSSION TITLE
How do i get a complete list of commands available in pybricks.

### DISCUSSION THREAD

**Discussion by CTFlorian** [pybricks/support#776] [Q&A] [answered]
Im looking for the native python commands and whats in each lego import module. 
What is the python version that pybricks uses.

**Comment by dlech** [comment, marked as answer]
Pybricks use [MicroPython](http://www.micropython.org/), not regular Python. And Pybricks is based on the minimal port of MicroPython, so it doesn't enable everything that some other MicroPython ports enable. There is complete documentation of all of the APIs available in the app and also at https://docs.pybricks.com
